docs/usage/20-provisioning-ingresses.adoc

```diff
@@ -148,6 +148,26 @@ kubernetes.dcos.io/dklb-config: |
 
 In the above representation, `<edgelb-pool-cpus>` is a floating-point number (e.g. `0.2`), and `<edgelb-pool-memory>` and `<edgelb-pool-size>` are integers (e.g. `512` and `3`, respectively).
 
+==== Customizing the EdgeLB load balancer instance placement constraints
+
+`dklb` supports customizing load balancer instance placement for the target EdgeLB Pool.
+The default is no constraint is specified. A custom value can be speficied using `.constraints` field.
```

Comment:
```suggestion
By default, no constraint is specified. A custom value can be specified using the `constraints` field.
```

Reply:
wth the spell checker didn't flag `speficied`?